samples/nfc/tag_reader/src/main.c

```diff
@@ -127,34 +320,19 @@ static void anticollision_completed(const struct st25r3911b_nfca_tag_info *tag_i
 
 		return;
 	}
-
-	err = st25r3911b_nfca_transfer_with_crc(&tx_buf, &rx_buf,
-						T2T_READ_CMD_FDT);
-
-	if (err) {
-		printk("Transaction error\n");
-	}
 }
 
 static void transfer_completed(const u8_t *data, size_t len, int err)
 {
 	if (err) {
 		printk("NFC Transfer error: %d\n", err);
-	} else {
-		printk("Transmission completed.\n");
+		return;
 	}
 
```

Comment:
```suggestion
if (t2t.state != T2T_IDLE) {
```

or return 0 from ```on_t2t_transfer_complete``` when t2t.state is idle. We don't want to print error when other tag types are being read.

Reply:
This sample now is not able to read other tag types,(for other tag types we do not try to transmit any data so this callback is only called when we read t2t ), so in this place I think this is not needed now.

Reply:
ok